website/src/word-panel.css.tsx

```diff
@@ -0,0 +1,32 @@
+import { style } from "@vanilla-extract/css"
+import sprinkles from "./sprinkles.css"
+
+export const wholePanel = style({
+    position: "sticky",
+    top: 150,
+})
+
+export const wordPanelButton = style({
+    lineHeight: 1.5,
+    boxSizing: "border-box",
+    border: "transparent",
+    cursor: "pointer",
+    fontSize: "16px",
+    borderRadius: "0.25rem",
+    ":before":{
+        display: "inline-block",
+        margin: "4px",
+    },
```

Comment:
I'm not certain this works? With [vanilla-extract](https://vanilla-extract.style/documentation/styling-api/) you normally need to use
```ts
selectors: {
  "&:before": {
    ...
  }
}
```

Reply:
Yeah, I wasn't quite sure about how to write this section. I must have left that part out.

Reply:
Just checked the docs again, and for simple pseudo-selectors you don't need to nest them under "selectors", so the code you wrote should work just fine.